internal/convert/cluster_config_test.go

```diff
@@ -27,6 +27,7 @@ import (
 
 func TestClusterConfig_PatchAutomationConfig(t *testing.T) {
 	fipsMode := true
+	defaultWriteConcernJ := true
```

Comment:
it would be great to update the e2e test to run an update command [e2e/cloud_manager/deploy_replica_set_test.go](https://github.com/mongodb/mongodb-atlas-cli/blob/master/test/e2e/cloud_manager/deploy_replica_set_test.go). Happy to move it to another PR.

Reply:
Good idea. Capturing in https://jira.mongodb.org/browse/CLOUDP-166012 for triage.